msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/entity/TopicEntityProviderImpl.java

```diff
@@ -375,7 +375,6 @@ public void deleteEntity(EntityReference ref, Map<String, Object> params) {
 				}
 
 				for (DiscussionForum forum : forums) {
-					if(forum.getDraft().equals(Boolean.FALSE)){
```

Comment:
No, If a person don't have permission won't see the Drafted forums/Forum topics. 
In order to see the forum feed firstly the person needs to be part of the site. Secondly, a student who is part of the site and some of the Forums/ Forums Topics are drafted even in this case  from the entitybroker interface he can't see the drafted Forums/Forum topics. It's just the person who drafted the Forums/ Forum Topics can see the Drafted info in the Entity Broker Feed